pages/doc/logging_drill_into_logs.md

```diff
@@ -0,0 +1,169 @@
+    
+   Click the eye icons of the related logs filters that you want to hide from the logs search query. To unhide a filter, you must click the eye-hide icon. You cannot remove or hide the `time range` filter.
+  
+1. In the **Alert Log Summary** panel, click **Go to Logs (Beta)**.
+    The Logs Browser opens in a new tab with the configurations from the **Related Logs** panel:
+    
+    * The search time window corresponds to the `time range` value.
+    * The search query contains the unhidden filters (with the eye icons).
+      ![The search query and the selected time window in the Logs Browser.](images/logs_drill_alert_search.png)
+
+<table style="width: 100%;">
+<tbody>
+<tr><td width="90%">&nbsp;</td><td width="10%"><a href="logging_overview.html"><img src="/images/to_top.png" alt="click for top of page"/></a></td></tr>
+</tbody>
+</table>
+
+## Drill into Logs from Traces
+
+If you have the **Logs** permission, you can drill into logs from the application status page and the Traces Browser.
+
+{% include note.html content="Even if logging is enabled for your environment, the drill-down from traces to logs might have to be enabled separately. Contact [technical support](wavefront_support_feedback.html#support)." %}
+
+### Application Status
+
+If you notice that a service on the application map, table view, or grid view of the [Application Status page](tracing_ui_overview.html) has a high error percentage, you can drill down into the related logs.
+
+{% include note.html content="You must have tagged the traces and the logs from the same applications and services with equivalent application and service tag values. If your traces and logs tags don't match, users with **Accounts** permissions can map the traces tags to logs tags. For details, see [Customize Logs Settings](logging_logs_settings.html)."%}
+
+* **From the Map View**
+  1. Select the time window of interest.
+  1. Click the service on the application map.
+  1. Select **View Logs (Beta)**.
+  ![A screenshot of a the UI once you click on a service with the view logs link highlighted.](images/logging_app_map_to_logs.png)
+* **From the Table View**
+  1. Select the time window of interest.
+  1. Click the ellipsis for the service.
+  1. Select **View Logs (Beta)**.
+  ![A screenshot of a the UI once you click vertical ellipsis on the table view](images/logging_table_view_to_logs.png)
+* **From the Grid View**
+  1. Select the time window of interest.
+  1. In a service tile, click **Actions**.
+  1. Select **View Logs (Beta)**.
+  ![A screenshot of a the UI once you click vertical ellipsis on the grid view](images/logging_grid_view_to_logs.png)
+
+The Logs Browser opens in a new tab with the following configurations:
+  
+* The search time window corresponds to the time window on the Application Status page.
+* The search query contains the corresponding include `service` and `application` tag filters.
```

Comment:
* The search query contains the corresponding `service` and `application` tag filters included.